css/style.css

```diff
@@ -376,6 +383,17 @@ li {
   display: flex;
   flex-direction: column;
   gap: 20px;
+  justify-content: center;
+  align-items: center;
+}
+
+form input {
+  padding: 12px;
+  width: 100%;
+}
+
+form textarea {
+  padding: 12px;
 }
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure that your design match that of the Figma template as much as possible. Kindly increase the width of the input fields.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/171876442-7ac67294-9b11-415e-9cf6-83e4f6428dbc.jpg)
Figma design
![figma](https://user-images.githubusercontent.com/85095702/171876486-3e5486b7-5718-45f8-95bb-b8aac693cfe2.png)